configure.ac

```diff
@@ -630,10 +652,11 @@ AC_DEFINE_UNQUOTED([PCMK__API_VERSION], ["$API_VERSION"],
 CIB_VERSION=$(latest_schema_version "xml")
 AC_SUBST(CIB_VERSION)
 
-# Add the schemas to the files that cause automake to re-run configure at next
-# make, so we re-detect the highest version
-AC_SUBST([CONFIG_STATUS_DEPENDENCIES],
-         [$(echo '$(wildcard $(top_srcdir)/xml/api/*.rng)')])
+# Re-run configure at next make if schema files change, to re-detect versions
+cib_schemas="$(schemas_for_make "xml")"
+api_schemas="$(schemas_for_make "xml/api")"
+CONFIG_STATUS_DEPENDENCIES="$cib_schemas $api_schemas"
+AC_SUBST(CONFIG_STATUS_DEPENDENCIES)
 
 
 dnl ==============================================
```

Comment:
I'm curious if this will help straighten out some of the weirdness I'm seeing in #3189, where it looks like after it runs "make check" it runs "make" again, and then there are errors.  I haven't figured out the errors yet, since I'm hoping the redundant build is the cause.